Apps/W1/APIV2/app/src/pages/APIV2FALocations.Page.al

```diff
@@ -0,0 +1,36 @@
+page 30097 "APIV2 - FA Locations"
```

Comment:
I do not need the FA classes at this moment so I think it can be part of other pull request if someone will need it. That also does not need to be added as a part since this is dictionary. Comparing to items you would need to add Item Category, Item Trancking etc. parts to each item and I believe you do not need it and you do not do it.

Reply:
It is about having a complete API, but you can leave it we can add it ourselves.